### DISCUSSION TITLE
API update files playlist

### DISCUSSION THREAD

**Discussion by jonasjersborg** [AzuraCast/AzuraCast#7140] [Q&A] [answered]
Hi all,
I'm new here so bare with me 😬
When i use the endpoint PUT /station/{station_id}/file/{id} i use the following json to add the song to a playlist.
But what are the weight here used for? I know about the playlists own weight for rotaion but the file weight is not documented or am i blind?
If i dont include the weight part it just gets a random weight in the 200-300 area.

    "playlists": [
        {
            "id": 4,
            "name": "default",
            "weight": 357
        }
    ],

**Comment by BusterNeece** [comment, marked as answer]
@jonasjersborg That's for sorting Sequential playlists. It's not used on other playlist orders. :)

**Reply by jonasjersborg** [reply]
So if i just send this, it would be fine for adding a file to a playlist
"playlists": [
    {
        "id": 4
    }
]

**Reply by BusterNeece** [reply]
It definitely should be!

**Reply by jonasjersborg** [reply]
Thank you!! 👍
